src/data-workspace/section-form/section-description.js

```diff
@@ -0,0 +1,24 @@
+import * as DOMPurify from 'dompurify'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './section.module.css'
+
+export const SectionDescription = ({ children }) => {
+    if (!children) {
+        return null
+    }
+    const html = DOMPurify.sanitize(children, {
+        ALLOWED_TAGS: ['a', 'b', 'strong', 'underline'],
+    })
+
+    return (
+        <div
+            className={styles.sectionDescription}
+            dangerouslySetInnerHTML={{ __html: html }}
+        ></div>
+    )
+}
+
+SectionDescription.propTypes = {
+    children: PropTypes.node.isRequired,
```

Comment:
```suggestion
    children: PropTypes.node,
```

Oh actually this will cause a prop warning if a section doesn't have a description configured, which may be common; maybe this shouldn't be required? (minor)

Reply:
yup, you're right - updated now